lib/Sema/CSSimplify.cpp

```diff
@@ -2060,17 +2060,13 @@ static bool isInPatternMatchingContext(ConstraintLocatorBuilder locator) {
 
   if (!path.empty()) {
     // Direct pattern matching between tuple pattern and tuple type.
```

Comment:
We should also update both comments =]

Reply:
Thanks.
I'm thinking of changing it to:
```
Direct pattern or sub-pattern matching of tuple element.
 e.g.`case .foo((a: 42, _)) = question`
```
Do you have any suggestions? 👀

Reply:
Sounds good!